roles/ansible-role-tests/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Clone stable-2.5 ansible
```

Comment:
Zuul seems to block this.

So 
1) You could do something like https://github.com/ansible-network/ansible-zuul-jobs/pull/9/files#diff-dbd9cd0ce7e9a5628770143f1488ff59R5
2) pip install ansible (which will give us latest stable, rather than head of the stable branch)